### PR TITLE
feat: add radar pipeline

### PR DESCRIPTION
**Pull request by scepter914**
## Description

Add radar pipeline.

- <https://github.com/autowarefoundation/autoware.universe/pull/4994> object merger queue size
- <https://github.com/autowarefoundation/autoware.universe/pull/4990> simple_object_merger の timeout handling 修正
- <https://github.com/autowarefoundation/autoware.universe/pull/4992> radar_object_fusion_to_detected_object のconfidence修正
- <https://github.com/autowarefoundation/autoware.universe/pull/4993> radar_track_msg_converter 速度座標変換 修正
- <https://github.com/autowarefoundation/autoware.universe/pull/5026> detection側でfar_objects merge 対応
- <https://github.com/autowarefoundation/autoware.universe/pull/5058> object merger 修正
- <https://github.com/autowarefoundation/autoware.universe/pull/5166> Camera-LiDAR-Radar fusion launcher fix

## Tests performed

Tests by rosbag

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml [hunk 1]
```diff
@@ -102,6 +105,48 @@
...
+
+    <!-- simple_cluster_merger -->
+    <group>
+      <include file="$(find-pkg-share cluster_merger)/launch/cluster_merger.launch.xml" if="$(var use_roi_based_cluster)">
+        <arg name="input/cluster0" value="euclidean_cluster/clusters"/>
+        <arg name="input/cluster1" value="roi_cluster/clusters"/>
+        <arg name="output/clusters" value="clusters"/>
+      </include>
+    </group>
+
```
Comment (badai-nguyen) [inline launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml:149]: @scepter914 I think this code is duplicated? It is already existed in. https://github.com/tier4/autoware.universe/blob/eec862948af2530c7f4229dd4909d464dbb64158/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml#L197-L228

I am sorry maybe it was my fault to put it after eculidean clustering.

**Comment by badai-nguyen** [comment]
@scepter914 I was my mistake in
https://github.com/tier4/autoware.universe/blob/eec862948af2530c7f4229dd4909d464dbb64158/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml#L191
that make `roi_cluster_fusion` disconnected with `shape_estimation`

Could you change back it to: 
`<arg name="output/clusters" value="clusters"/>`
I am sorry for that.

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml (hunk 1 above)
Comment (scepter914) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml:149]: fixed at [af3d958](https://github.com/tier4/autoware.universe/pull/892/commits/af3d9583285ed9e7cc3eda3ca9ffd4256b3e9aeb)

Comment (badai-nguyen) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml:149]: @scepter914 thank you. It works now. Could but it is better to remove https://github.com/tier4/autoware.universe/blob/af3d9583285ed9e7cc3eda3ca9ffd4256b3e9aeb/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml#L197-L204 

this is also duplicated with https://github.com/tier4/autoware.universe/blob/af3d9583285ed9e7cc3eda3ca9ffd4256b3e9aeb/launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml#L141-L148. 

:pray:

Comment (scepter914) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml:149]: fixed at [7f88d81](https://github.com/tier4/autoware.universe/pull/892/commits/7f88d81e5846fdabbe9af571680a9268207c98a9)

**Review by badai-nguyen — APPROVED** [review]
LGTM

**Pull request merged**